docs/changelog.md

```diff
@@ -4,17 +4,24 @@
 
 - :new: new: DICOMweb client for WADO (Web Access to DICOM Objects)/QIDO (Query based on ID for DICOM Objects)/STOW 
   (Store Over the Web) and a CLI is available in [DicomWebClient](https://github.com/NVIDIA/clara-dicom-adapter/tree/main/src/DicomWebClient).
-- :new: new: New REST API to trigger a new inference request is now avilable based on the specs defined by the 
+- :new: new: New REST API to trigger a new inference request is now available based on the specs defined by the 
   American College of Radiology (ACR). Please refer to the API Documentation for more information.
+- :warning: Clara CLI 0.8.1 is required.  Please reference to [compatibility](./setup/compatibility.md) matrix for additional information.
```

Comment:
"Please reference to" -> "Refer to"

no need to say please in this case, also refer (verb) vs reference (noun)

Reply:
Fixed typo 🙏